src/lib.rs

```diff
@@ -1,11 +1,11 @@
 use borsh::{BorshDeserialize, BorshSerialize};
-use near_bindgen::{env, metadata, near_bindgen};
+use near_sdk::{env, metadata, near_bindgen};
 use std::collections::HashMap;
 
 #[global_allocator]
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
-metadata!{
+metadata! {
```

Comment:
You also don't need metadata. Please remove it

Reply:
I removed the `metadata` wrapper but kept the content within – is that what you meant? The tests did not pass if I removed the whole block.

Reply:
Yes. Just remove the macro